### DISCUSSION TITLE
Documentation -Web Api users tab is missing

### DISCUSSION THREAD

**Discussion by cntrvsy** [pocketbase/pocketbase#1122] [Q&A] [answered]
Hello, it has been a while since I interacted with pocketbase and I wanted to know where this information was moved to. I'm sorry if this a trivial question I am new to the coding space and my navigation around documentation is not the best.

**Comment by ganigeorgiev** [comment, marked as answer]
With PocketBase v0.8.0 there are is no longer `User` model since it was merged with the `profiles` collection per #376.

More info on the changes for v0.8.0 and a migration guide to v0.7.x could be found in https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md#v080.

In short, all auth-related actions are now under the `/api/collections/*/` endpoint. You could find detailed definitions either in the API Preview for your auth collection OR in https://pocketbase.io/docs/api-records/#auth-record-actions.
